app/views/rooms/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<%- @skip_links = {filters: "Filters"} %>
```

Comment:
keeping erb in .erb files and haml in .haml files keeps the reading much easier. Can you change this to erb code syntax

Reply:
I am not clear on what you mean. The code you referenced is erb and is located in the erb template. Can you clarify? Thanks!

Reply:
this was a mis-read on my part. Looks good